db/sqlinterfaces.c

```diff
@@ -6747,6 +6752,10 @@ static int internal_peer_check(struct sqlclntstate *a)
 {
     return 0;
 }
+void * internal_get_authdata(struct sqlclntstate *a)
```

Comment:
I guess you need noop wrapper for fastsql as well..

Reply:
its in plugin-branch eventbuf_fix